backend/src/test/java/com/ody/common/RedisTestContainersConfig.java

```diff
@@ -0,0 +1,35 @@
+package com.ody.common;
+
+import java.time.Duration;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.data.redis.connection.RedisConnectionFactory;
+import org.springframework.data.redis.connection.lettuce.LettuceConnectionFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@TestConfiguration
+@Testcontainers
+public class RedisTestContainersConfig {
+    private static final int REDIS_PORT = 6379;
```

Comment:
```suggestion

    private static final int REDIS_PORT = 6379;
```

저도 개행을 잘 못챙기는 편이지만.. ㅎㅎ 그래도 보여서 남겨놓아요 😅